_maps/configs/pgf_crying_sun.json

```diff
@@ -37,7 +37,7 @@
 			"outfit": "/datum/outfit/job/gezena/assistant",
 			"slots": 2
 		},
-		"Marine Sergeant": {
+		"Marine Lieutenant": {
```

Comment:
Make this guy an officer maybe? Or was that discussed already?
He _probably_ should be one, cause command.

Reply:
Definitely something I'll run past Latency. I just left it as it was originally, so I'd say it depends if it was a part of the original intent or just something that got overlooked.